client/my-sites/marketing/connections/service.jsx

```diff
@@ -595,7 +595,7 @@ export class SharingService extends Component {
 					/>
 					<Notice isCompact status="is-error" className="sharing-service__unsupported">
 						{ this.props.translate(
-							'Twitter is no longer supported. {{a}}Learn more about this{{/a}}',
+							'X (Twitter) is {{a}}no longer supported{{/a}}. You can still use our quick {{a2}}manual sharing{{/a2}} feature from the post editor to share to it!',
```

Comment:
I just put it there to make it sound better, because manual sharing doesn't sound like an interesting feature, because it's manual. But I can remove it

Reply:
Fair enough. Then you can ignore that suggestion. Thank you.

Reply:
I agree that it's not the best sounding sentence, I'm not sure how can we make it better! :D